cmake/FindICU.cmake

```diff
@@ -0,0 +1,428 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
What uses this?

Reply:
It is copied from latest cmake repo with minor modification for this build system to work with older version cmake, say, CMake 3.5 on Ubuntu 16.04. ICU is needed of a third party library for i18n.